asciidoc-confluence-publisher-converter/src/test/java/org/sahli/asciidoc/confluence/publisher/converter/FolderBasedAsciidocPagesStructureProviderTest.java

```diff
@@ -0,0 +1,56 @@
+package org.sahli.asciidoc.confluence.publisher.converter;
+
+import org.junit.Test;
+import org.sahli.asciidoc.confluence.publisher.converter.AsciidocPagesStructureProvider.AsciidocPage;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.not;
+import static org.hamcrest.Matchers.nullValue;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.mock;
+
+public class FolderBasedAsciidocPagesStructureProviderTest {
+
+    private static AsciidocPage NON_EXISTING_ASCIIDOC_PAGE = mock(AsciidocPage.class);
+
+    @Test
+    public void structure_nestedStructure_returnsAsciidocPagesStructureWithAllNonIncludeAdocFiles() throws Exception {
```

Comment:
Typo in `AllNonIncludeAdocFiles`, `d` is missing for `Include`.

Reply:
In the code, we use "include file" as the concept, not "included files" (e.g. see `INCLUDE_FILE_PREFIX`)